classes/class-wc-connect-jetpack.php

```diff
@@ -47,14 +45,7 @@ public static function is_development_mode() {
 		 * @return bool
 		 */
 		public static function is_active() {
-			if ( defined( 'WOOCOMMERCE_SERVICES_LOCAL_TEST_MODE' ) && WOOCOMMERCE_SERVICES_LOCAL_TEST_MODE ) {
-				return true;
-			}
-			if ( method_exists( 'Jetpack', 'is_active' ) ) {
-				return Jetpack::is_active();
-			}
-
-			return false;
+			return ! empty ( self::get_access_token( true ) );
```

Comment:
The code only checks for connection owner's "user token". I'd recommend to also check for "blog token" in case of some connection issues that get the "blog token" removed, but leave "user tokens" intact.

```suggestion
			return self::get_connection_manager()->is_connected() && self::get_connection_manager()->has_connected_owner();
```